pkg/gator/test/test.go

```diff
@@ -26,10 +26,20 @@ func init() {
 	}
 }
 
-func Test(objs []*unstructured.Unstructured) (*GatorResponses, error) {
-	// create the client
+func Test(objs []*unstructured.Unstructured, mutators ...TestOptionMutator) (*GatorResponses, error) {
```

Comment:
Can we just call these `Options` instead of `TestOptionMutator`? `Mutator` is overloaded here